packages/react-instantsearch-dom/src/components/__tests__/SmartSort.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import renderer from 'react-test-renderer';
+import SmartSort from '../SmartSort';
+
+const textComponent = ({ isSmartSorted }) => (
+  <div>
+    {isSmartSorted
+      ? 'We are filtering unrelevant results'
+      : 'We are not filtering unrelevant results'}
+  </div>
+);
+
+const buttonTextComponent = ({ isSmartSorted }) => (
+  <div>{isSmartSorted ? 'See all results' : 'See relevant results'}</div>
+);
+
+textComponent.propTypes = {
```

Comment:
is proptypes still required in a ts file? could you have annotated directly in the component with ts to have a smaller footprint?

Reply:
Yeah I can, should I create a type or just inline it as `({ isSmartSorted }: { isSmartSorted: boolean }) => ()` ?

Reply:
export the prop type from the component as SmartSortComponentProps and use it here maybe?